homework-classes/App.js

```diff
@@ -20,11 +20,27 @@ class App {
     const root = document.getElementById('root');
     const header = Util.createAndAppend('header', root, { class: 'header' });
     this.mainContainer = Util.createAndAppend('div', root, { id: 'container' });
-
+    Util.createAndAppend('h2', header, {
+      class: 'heading-two',
+      text: 'HYF Repositories:',
+    });
+    const selectElement = Util.createAndAppend('select', header);
     try {
       const repos = await Util.fetchJSON(url);
       this.repos = repos.map(repo => new Repository(repo));
       // TODO: add your own code here
+
+      repos.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
There is a discrepancy between what you sort and what you use to display the repo details: the select box shows the `alumni` repo, but the details are for the `AngularJS` repo.

![image](https://user-images.githubusercontent.com/2788771/62779906-b8463800-bab4-11e9-83cd-abf425c13b97.png)